samples/maps/demo/basic-map/demo.js

```diff
@@ -0,0 +1,63 @@
+(async () => {
+
+    const topology = await fetch(
+        'https://code.highcharts.com/mapdata/custom/world.topo.json'
+    ).then(response => response.json());
+
+    Highcharts.mapChart('container', {
+        chart: {
+            map: topology
+        },
+
+        title: {
+            text: 'Life expectancy in the world (2021)'
+        },
+
+
+        credits: {
+            href: 'https://data.worldbank.org',
+            mapText: ' Data source: The World Bank'
+
+        },
+        mapNavigation: {
+            enabled: true,
+            buttonOptions: {
+                verticalAlign: 'bottom'
+            }
+        },
+
+        colorAxis: {
+            min: 60
+        },
+
+        data: {
+            csv: document.getElementById('csv').innerText,
+            seriesMapping: [{
+                code: 1,
+                value: 2
+            }]
+        },
+
+        tooltip: {
+            pointFormat: '{point.name}: <b>{point.value}</b> years'
```

Comment:
Number in tooltip should likely be truncated a bit.
![image](https://github.com/highcharts/highcharts/assets/7478772/577793eb-4a3e-497e-bfc3-98dc971f314b)

Had to [look up](https://www.highcharts.com/docs/chart-concepts/templating) how to do it with a format string; looks like this does the trick:
```js
pointFormat: '{point.name}: <b>{point.value:.1f} </b> years'
```